src/biogeophys/LunaMod.F90

```diff
@@ -1048,7 +1069,7 @@ subroutine Nitrogen_investments (KcKjFlag, FNCa, Nlc, forc_pbot10, relh10, &
   theta = 0.292_r8 / (1.0_r8 + 0.076_r8 / (Nlc * Cb))
   ELTRNabsorb = theta * PARi10
   Jmaxb0act = params_inst%jmaxb0 * FNCa * Fj
-  Jmax = Jmaxb0act + JmaxCoef * ELTRNabsorb
+  Jmax = Jmaxb0act + JmaxCoef * ELTRNabsorb * o3coefjmaxsun
```

Comment:
You are right, it is never referenced. At that point, I understood that LUNA is not run for different light modes ('shaded', 'sun') explicitly, I should have removed it. That separation is only made later on. I have not fully grasped how LUNA fits into a canopy level specific light intensity treatment. Hence, the differentiation between "sha" and "sun" is a relic.

Reply:
This is in fact some open issue with my own code that bothers me. I would think LUNA should compute Jmax/Vcmax (and also gsto and photsynthesis) for different canopy levels rather than scaling sun/sha in accordance to an exponential attenuation function. But I might be mistaken there.

Anyway, I found some buggy (or shady) behavior in derived "shaded" variables in my test runs.